### PR TITLE
working OPDS JSON Schema (tested with NodeJS r2-streamer-js)

### PR DESCRIPTION
**Pull request by danielweck**
see https://github.com/edrlab/r2-streamer-js/issues/32

Changes explained:

* JSON files formatted by VSCode, same as with my other (merged) PR for webpub manifest.
* fixed fatal JSON errors: unnecessary leading commas and curly braces
* readium.org webpub manifest URL is now secure HTTPS

### CODE REVIEW

File: schema/properties.schema.json [hunk 1]
```diff
@@ -19,23 +19,193 @@
...
-            "ILS", "INR", "IQD", "IRR", "ISK", "JMD", "JOD", "JPY", "KES", "KGS", "KHR", "KMF", "KPW", 
-            "KRW", "KWD", "KYD", "KZT", "LAK", "LBP", "LKR", "LRD", "LSL", "LYD", "MAD", "MDL", "MGA", 
-            "MKD", "MMK", "MNT", "MOP", "MRU", "MUR", "MVR", "MWK", "MXN", "MXV", "MYR", "MZN", "NAD", 
-            "NGN", "NIO", "NOK", "NPR", "NZD", "OMR", "PAB", "PEN", "PGK", "PHP", "PKR", "PLN", "PYG", 
-            "QAR", "RON", "RSD", "RUB", "RWF", "SAR", "SBD", "SCR", "SDG", "SEK", "SGD", "SHP", "SLL", 
-            "SOS", "SRD", "SSP", "STN", "SVC", "SYP", "SZL", "THB", "TJS", "TMT", "TND", "TOP", "TRY", 
-            "TTD", "TWD", "TZS", "UAH", "UGX", "USD", "USN", "UYI", "UYU", "UZS", "VEF", "VES", "VND", 
-            "VUV", "WST", "XAF", "XAG", "XAU", "XBA", "XBB", "XBC", "XBD", "XCD", "XDR", "XOF", "XPD", 
-            "XPF", "XPT", "XSU", "XTS", "XUA", "XXX", "YER", "ZAR", "ZMW", "ZWL"]
+          "enum": [
```
Comment (HadrienGardeur) [inline schema/properties.schema.json]: Do we really need that one?

Comment (danielweck) [reply schema/properties.schema.json]: Do you mean the reformat? If so, well I could try to tweak my editor's settings (VSCode), but I would rather not :)
Alternatively, I can copy / paste the original text selection over the reformatted piece.

Comment (HadrienGardeur) [reply schema/properties.schema.json]: It's OK to have an enum with one value per line when we have a maximum of 5-10 values, but this feels a little overwhelming.
If you don't mind pasting the original text selection, I'd be much more comfortable accepting this PR.

Comment (danielweck) [reply schema/properties.schema.json]: done

**Pull request merged**